Tests/BackendIntegrationTests/OtherIntegrationTests.swift

```diff
@@ -31,6 +31,74 @@ class OtherIntegrationTests: BaseBackendIntegrationTests {
         expect(info.isComputedOffline) == false
     }
 
+    func testGetCustomerInfoMultipleTimesInParallel() async throws {
```

Comment:
maybe we could have another test that also verifies the behavior when you call `customerInfo()` right after the response came back? i.e.: verifying that we're still respecting the TTL

Reply:
Hmm yeah I don't think we have any _integration_ test for that, let me add it.